README.md

```diff
@@ -7,35 +7,42 @@ Usage
 
 Quick example (docs coming soon):
 
-    json_value * arr = json_array_new(0);
-    json_array_push(arr, json_string_new("Hello world!"));
-    json_array_push(arr, json_integer_new(128));
+```c
+json_value * arr = json_array_new(0);
+json_array_push(arr, json_string_new("Hello world!"));
+json_array_push(arr, json_integer_new(128));
 
-    char * buf = malloc(json_measure(arr));
-    json_serialize(buf, arr);
+char * buf = malloc(json_measure(arr));
+json_serialize(buf, arr);
 
-    printf("%s\n", buf);
+printf("%s\n", buf);
+```
 
+```
 > [ "Hello world!", 128 ]
+```
```

Comment:
Might as well make this `json` syntax highlighted while you're at it.

Reply:
good call, done!